src/modules/localStorage.js

```diff
@@ -0,0 +1,13 @@
+const addListToLocalStorage = (listToStore) => {
+  localStorage.setItem('taskList', JSON.stringify(listToStore));
+};
+
+const loadListFromLocalStorage = () => {
+  const listToLoad = JSON.parse(localStorage.getItem('taskList'));
```

Comment:
```suggestion
  const listToLoad = JSON.parse(localStorage.getItem('taskList'));
``` 
I think it would be good to avoid using conditionals when possible. You could use a shorthand conditional by defining **listToLoad** = JSON.parse(localStorage.getItem('taskList')) || (or) [] (empty array);
![image](https://user-images.githubusercontent.com/105621110/193683789-f2551038-d482-4165-899a-7351d2bad804.png)